src/Text.mo

```diff
@@ -19,19 +28,36 @@ import Prim "mo:⛔";
 
 module {
 
-  /// Text values.
+  /// The type corresponding to primitive `Text` values.
+  ///
+  /// ```motoko
+  /// let hello = "Hello!";
+  /// let emoji = "👋";
+  /// let concat = hello # " " # emoji; // "Hello! 👋"
+  /// ```
   public type Text = Prim.Types.Text;
 
-  /// Conversion.
-  /// Returns the text value of size 1 containing the single character `c`.
+  /// Converts the given `Char` to a `Text` value.
+  ///
+  /// ```motoko include=import
+  /// let text = Text.fromChar('A'); // "A"
+  /// ```
   public let fromChar : (c : Char) -> Text = Prim.charToText;
 
-  /// Conversion.
-  /// Creates an iterator that traverses the characters of the text `t`.
+  /// Iterates over each `Char` value in the given `Text`.
+  ///
+  /// Equivalent to calling the `t.chars()` method where `t` is a `Text` value.
+  ///
+  /// ```motoko include=import
+  /// let iter = Text.toIter("abc"); // iterator ('a', 'b', 'c')
```

Comment:
I've been using the iterators in for loops in examples. I think it gives a better example of how to use `toIter`. What do you think?

Reply:
Yep, good idea; updated.